python/tests/test_account.py

```diff
@@ -1524,6 +1524,59 @@ def test_send_receive_locations(self, acfactory, lp):
         locations3 = chat2.get_locations(contact=contact)
         assert not locations3
 
+    def test_autodelete_timer(self, acfactory, lp):
+        ac1, ac2 = acfactory.get_two_online_accounts()
+
+        lp.sec("ac1: create chat with ac2")
+        chat1 = self.get_chat(ac1, ac2)
+        chat2 = self.get_chat(ac2, ac1)
+
+        lp.sec("ac1: set autodelete timer to 60")
+        chat1.set_autodelete_timer(60)
+
+        lp.sec("ac1: check that autodelete timer is set for chat")
+        assert chat1.get_autodelete_timer() == 60
+        chat1_summary = chat1.get_summary()
+        assert chat1_summary["autodelete_timer"] == 60
+
+        lp.sec("ac2: receive system message about autodelete timer modification")
+        ac2._evtracker.get_matching("DC_EVENT_CHAT_AUTODELETE_TIMER_MODIFIED")
+        incoming_message_event1 = ac2._evtracker.get_matching("DC_EVENT_INCOMING_MSG")
+        system_message1 = ac2.get_message_by_id(incoming_message_event1.data2)
+        assert chat2.get_autodelete_timer() == 60
+        assert system_message1.is_system_message()
+        assert "Autodelete timer: 60\n" in system_message1.get_message_info()
```

Comment:
please use get_message_info().strip() and don't match the "\n".  we don't care about whitespace here.